packages/@aws-cdk-testing/framework-integ/test/aws-cloudfront-origins/test/integ.origin-group-selection-criteria.ts

```diff
@@ -0,0 +1,35 @@
+import * as cloudfront from 'aws-cdk-lib/aws-cloudfront';
+import * as cdk from 'aws-cdk-lib';
+import * as origins from 'aws-cdk-lib/aws-cloudfront-origins';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { CfnChannelGroup } from 'aws-cdk-lib/aws-mediapackagev2';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'cloudfront-origin-group-selection-criteria');
+
+const channelGroup = new CfnChannelGroup(stack, 'cg1', {
+  channelGroupName: 'channelGroup1',
+});
+const channelGroup2 = new CfnChannelGroup(stack, 'cg2', {
+  channelGroupName: 'channelGroup2',
+});
+
+const originGroup = new origins.OriginGroup({
+  primaryOrigin: new origins.HttpOrigin(channelGroup.attrEgressDomain),
+  fallbackOrigin: new origins.HttpOrigin(channelGroup2.attrEgressDomain),
+  fallbackStatusCodes: [404],
+  selectionCriteria: cloudfront.OriginSelectionCriteria.MEDIA_QUALITY_BASED,
+});
+
+new cloudfront.Distribution(stack, 'Distribution', {
+  defaultBehavior: { origin: originGroup },
+  additionalBehaviors: {
+    '/video': {
+      origin: originGroup,
+    },
+  },
+});
+
+new IntegTest(app, 'origin-group-selection-criteria', {
```

Comment:
Could we add an assertion here to call `getDistributionConfig` and check the value of `SelectionCriteria` is as expected? An example integ test you can use for reference: https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk-testing/framework-integ/test/aws-cloudfront-origins/test/integ.s3-origin-oac.ts#L25

Reply:
I have tried to do this but `SelectionCriteria` isn't returned at the moment - I wonder if the aws-sdk needs updating? Apologies, not deep dived this part of CDK yet

Reply:
@gracelu0 - I've taken another look, and dived into the logs in CloudWatch - but the JSON body returned on the call doesn't include the `SelectionCriteria` - any advice on trying to get it? Am I missing something?